protected/humhub/modules/topic/widgets/TopicLabel.php

```diff
@@ -9,21 +9,23 @@
 
 namespace humhub\modules\topic\widgets;
 
-use humhub\modules\content\models\ContentTag;
+use humhub\modules\content\components\ContentContainerActiveRecord;
 use humhub\modules\topic\models\Topic;
 use humhub\widgets\Label;
 use humhub\widgets\Link;
-use yii\helpers\Html;
 
 class TopicLabel extends Label
 {
     /**
      * @param Topic $topic
      * @return $this
      */
-    public static function forTopic(Topic $topic)
+    public static function forTopic(Topic $topic, ?ContentContainerActiveRecord $contentContainer)
```

Comment:
I'll receive an internal error in the Wiki Module, when specify a topic.


![image](https://github.com/user-attachments/assets/348b64c1-04f5-4b7f-b2d0-86e91090a840)

![image](https://github.com/user-attachments/assets/28c6ef27-df71-4539-b8a4-b6cf32e62aee)